chemreg/compound/models.py

```diff
@@ -103,3 +87,32 @@ class QueryStructureType(CommonInfo):
 
     def __str__(self):
         return self.label
+
+
+def get_illdefined_qst():
+    qst, created = QueryStructureType.objects.get_or_create(
+        name="ill-defined",
+        defaults={"label": "Ill defined", "short_description": "Ill defined"},
+    )
+    if created:
+        qst.save()
+    return qst.pk
```

Comment:
Is this valid business logic? I don't see a default like this being requested in the ticket.

Reply:
Yes, that relationship is defined here with the original model:
https://github.com/Chemical-Curation/chemcurator_django/issues/5